keyboards/handwired/onekey/keymaps/i2c_scanner/keymap.c

```diff
@@ -10,11 +10,11 @@
 #    pragma message("ChibiOS is currently 'best effort' and might not report accurate results")
 
 i2c_status_t i2c_start_bodge(uint8_t address, uint16_t timeout) {
-    i2c_start(address);
+    i2c_start(address, timeout);
```

Comment:
This can just be removed, because the existing ChibiOS implementation of `i2c_start()` is useless (and everything that is done there is done inside `i2c_readReg()` again).